Makefile

```diff
@@ -42,6 +42,27 @@ DBUS_POLICY_FILES = \
 
 build: mender
 
+## Hardcoded prototype
+CXXFLAGS = \
+	-I . \
+	-I vendor/
+
+LDFLAGS = \
+	-L common/json/ \
+	-l json
```

Comment:
> LDFLAGS
    Extra flags to give to compilers when they are supposed to invoke the linker, ‘ld’, such as -L. Libraries (-lfoo) should be added to the LDLIBS variable instead.

Might as well stick to the above.

Reply:
Ah, wait. This is from the hard-coded code in `Makefile` that I removed when actually introducing autotools. So no-op here :see_no_evil: